projects/extension/sql/idempotent/013-vectorizer-api.sql

```diff
@@ -31,6 +31,7 @@ create or replace function ai.create_vectorizer
 , queue_table pg_catalog.name default null
 , grant_to pg_catalog.name[] default ai.grant_to()
 , enqueue_existing pg_catalog.bool default true
+, loader pg_catalog.jsonb default null
```

Comment:
We're not using this right now right?

Reply:
we're using it to perform the validation of the chunking specified.

Reply:
https://github.com/timescale/pgai/blob/adol/tweak-chunking-for-s3/projects/extension/sql/idempotent/013-vectorizer-api.sql#L74-L76